Live-Snap/Live-Snap/Controllers/ExportWallpaperViewController.swift

```diff
@@ -101,10 +101,12 @@ class ExportWallpaperViewController: UIViewController, PHLivePhotoViewDelegate {
     }
     
     func livePhotoView(_ livePhotoView: PHLivePhotoView, didEndPlaybackWith playbackStyle: PHLivePhotoViewPlaybackStyle) {
-        view.addSubview(toolBar)
-        view.addSubview(forceTouchNotifierLabel)
-        animateToolBarAndLivePhotoView()
-        view.isUserInteractionEnabled = true
+        if !toolBar.isDescendant(of: view) {
```

Comment:
instead of doing this you should add the toolbar as a subview during viewdidload and set isHidden on it to true, then here check if its hidden instead of checking if its a descendant